Source/Diagnostics/Average.cpp

```diff
@@ -0,0 +1,55 @@
+#include "Average.H"
+
+using namespace amrex;
+
+AMREX_GPU_HOST_DEVICE
+AMREX_FORCE_INLINE
+Real Average::ToCellCenter ( Array4<Real const> const& mf_in_arr,
+                             const IntVect stag,
+                             int i,
+                             int j,
+                             int k,
```

Comment:
```suggestion
                             int const i,
                             int const j,
                             int const k,
```
these are passed by value, so adding `const` does not necessarily help with performance, but it clarifies the intent to other developers.